bentoml/saved_bundle/bundler.py

```diff
@@ -142,6 +142,17 @@ def save_to_dir(bento_service, path, version=None, silent=False):
             )
         )
 
+    # copy custom web_static_content if enabled
+    if bento_service.web_static_content:
+        src_web_static_content_dir = os.path.join(
+            os.getcwd(), bento_service.web_static_content
```

Comment:
add a `os.path.isdir(src_web_static_content_dir)` check here? , would be great to also support absolute path

Reply:
Added, it already supports absolute path. `os.path.join` is very smart!